client/state/application/actions.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Internal dependencies
+ */
+import { CONNECTION_LOST, CONNECTION_RESTORED } from 'state/action-types';
+import { warningNotice, successNotice, removeNotice } from 'state/notices/actions';
+import i18n from 'lib/mixins/i18n';
+
+export function connectionLost() {
+	return ( dispatch ) => {
+		dispatch( removeNotice( 'connectionRestored' ) );
+		dispatch( warningNotice(
+			i18n.translate( 'Not connected. Some information may be out of sync.' )
```

Comment:
Some context may be helpful for the translators, as they only see this string. Like `Message displayed when internet connection is lost`.